scene/gui/text_edit.cpp

```diff
@@ -8394,6 +8448,8 @@ TextEdit::TextEdit(const String &p_placeholder) {
 
 	v_scroll->connect("scrolling", callable_mp(this, &TextEdit::_v_scroll_input));
 
+	connect("lines_edited_from", callable_mp(this, &TextEdit::_clear_syntax_highlighting_cache).unbind(2));
```

Comment:
i this way of connecting it to the signal is easier to follow and if there is a new function added that edits lines, calling `_clear_syntax_highlighting_cache` doesn't have to be considered, less burden on a programmer adding functions

Reply:
I think its more clear if it is directly called rather than through a signal, and there shouldn't be anything that changes about editing lines that doesn't go through `_clear()` or `_text_changed()`, since both `_base_remove_text` and `_base_insert_text` call `_text_changed`.
Also if `set_block_signals` is used, it shouldn't disrupt something internal like clearing the syntax highlighting cache.

Reply:
i understand, that makes sense
i had no idea about `set_block_signals()` or `_text_changed()` i see now